### DISCUSSION TITLE
QRZ callbook lookup setup for the CloudLog subscription service?

### DISCUSSION THREAD

**Discussion by pacartwright** [magicbug/Cloudlog#1661] [General]
I would like Cloudlog to do a QRZ look up and pull in the QTH data for the QSO. How do I set this up using the paid subscription service?

**Comment by magicbug** [comment]
If your using paid its already setup for you for QRZ just type a callsign in and tab out the field :)

**Reply by pacartwright** [reply]
Thanks for the reply. Currently, I am not entering the qso by hand. I am using the ADIFpush to upload the QSO from the FT8 log. I guess I was wanting to have it fill in any missing information from QRZ when it is uploaded to CloudLog.  Is this currently possible?
